### DISCUSSION TITLE
Sorting of secret Table in netbox frontend view

### DISCUSSION THREAD

**Discussion by greinick** [DanSheps/netbox-secretstore#41] [Q&A]
Hi, may be thats an netbox issue, but I just see it in the netbox-secretstore plugin:

When clicking on the "assigned object" table row, the table is not sorted, but shows an error. The other lables work and the table gets sorted accordingly.

The error is:

<class 'django.core.exceptions.FieldError'>

Field 'assigned_object' does not generate an automatic reverse relation and therefore cannot be used for reverse querying. If it is a GenericForeignKey, consider adding a GenericRelation.

Python version: 3.8.10
NetBox version: 3.0.7

Any suggestions to check what's going wrong? Thanks for the plugin and your feedback.

**Comment by DanSheps** [comment]
Please open a issue for this.
